centrallix/expression/exp_functions.c

```diff
@@ -4466,25 +4503,42 @@ int exp_fn_argon2id(pExpression tree, pParamObjects objlist, pExpression passwor
 
 int exp_internal_DefineFunctions()
     {
-
-	/** Function list for EXPR_N_FUNCTION nodes **/
-	xhAdd(&EXP.Functions, "getdate", (char*)exp_fn_getdate);
+	/** Initialize library. **/
+	ca_init();
+	
+	/** Function list for EXPR_N_FUNCTION nodes. **/
+	
+	/** General. **/
```

Comment:
I really like the neater organization of the function names here, but let's defer this nice fix until after the UTF8 branch is merged in.  I'm aware of some changes in the order here that complicate the merge of that branch, making it hard to see what is different, and this will complicate that.

In the end, it might be better to have a name/function table of some sort (statically allocated), and then DefineFunctions can iterate over that to register them with xhAdd().

Reply:
Ahhhh, that makes sense. Change reverted.

My only change to this code section should now be the addition of the `metaphone()`, `ln()`, and `logn()` functions.